### PR TITLE
Update service-task.md

### PR DESCRIPTION
**Pull request by FabioBentoLuiz**
The casting with Long will fail on the step (link below) where the generic form screenshot sugests seting "amount" as Integer.
https://docs.camunda.org/get-started/quick-start/user-task/#work-on-the-task

### CODE REVIEW

File: get-started/content/quick-start/service-task.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ public class ChargeCardWorker {
 
           // Get a process variable
           String item = (String) externalTask.getVariable("item");
```
Comment (mboskamp) [inline get-started/content/quick-start/service-task.md:186]:
```suggestion
          String item = externalTask.getVariable("item");
```

File: get-started/content/quick-start/service-task.md [hunk 2]
```diff
@@ -184,7 +184,7 @@ public class ChargeCardWorker {
...
-          Long amount = (Long) externalTask.getVariable("amount");
+          Integer amount = (Integer) externalTask.getVariable("amount");
```
Comment (mboskamp) [inline get-started/content/quick-start/service-task.md:187]:
```suggestion
          Integer amount = externalTask.getVariable("amount");
```

**Review by mboskamp — CHANGES_REQUESTED** [review]
Hi @FabioBentoLuiz,

Thanks for the PR. You are right, it should be `Integer`. The corresponding [example repository](https://github.com/camunda/camunda-get-started-quickstart/blob/1f4f17c84d3a286aa9a648124bc7fbbae08f0f7f/java/charge-card-worker/src/main/java/org/camunda/bpm/getstarted/chargecard/ChargeCardWorker.java#L42) also uses `Integer` over `Long`.

I don't think the casts are even necessary. Let's remove them. This is then also consistent with the example. What do you think?

File: get-started/content/quick-start/service-task.md (hunk 2 above)
Comment (FabioBentoLuiz) [reply get-started/content/quick-start/service-task.md:187]: Hi @mboskamp,
Some steps later on the Get Started section (https://docs.camunda.org/get-started/quick-start/decision-automation/#inspect-using-cockpit-and-tasklist), there are two screenshots setting the amount as Long to start new process instances.

It is better updating them to use Integer as well, right?

Comment (mboskamp) [reply get-started/content/quick-start/service-task.md:187]: Oh, nice catch. We can take care of updating the screenshots with the next release. For now, let's merge the code snippet changes first.

**Pull request merged**